### PR TITLE
chore: add sdk_version to the network requests

### PR DESCRIPTION
**Pull request by duyhungtnn**
# Changes
- [x] add sdk_version to the `get_evaluations` request
- [x] add sdk_version to the `register_events` request

### CODE REVIEW

**Comment by duyhungtnn** [comment]
should close #17

File: Bucketeer/Sources/Internal/Remote/RequestBody/GetEvaluationsRequestBody.swift [hunk 1]
```diff
@@ -6,4 +6,7 @@ struct GetEvaluationsRequestBody: Codable {
     let userEvaluationsId: String
     let sourceId: SourceID
     let userEvaluationCondition: UserEvaluationCondition
+    // noted: we didn't set the default value to prevent the warning about Decodeable will not work
```
Comment (cre8ivejp) [inline Bucketeer/Sources/Internal/Remote/RequestBody/GetEvaluationsRequestBody.swift]: For this case, it's okay to only leave this comment on the PR.
No need to comment on the code.

Comment (duyhungtnn) [reply Bucketeer/Sources/Internal/Remote/RequestBody/GetEvaluationsRequestBody.swift]: sure, let me remove it

Comment (duyhungtnn) [reply Bucketeer/Sources/Internal/Remote/RequestBody/GetEvaluationsRequestBody.swift]: removed in https://github.com/bucketeer-io/ios-client-sdk/pull/48/commits/0eb8b43c08c7bc4a9feda77b9f42fcf138508110

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**